packages/sn-client-core/src/Repository/Upload.ts

```diff
@@ -74,20 +75,20 @@ export class Upload {
    * @param {File} file The File object
    * @param {Repository} repo The sensenet Repository
    */
-  public static isChunkedUploadNeeded(file: File, repo: Repository): boolean {
+  public isChunkedUploadNeeded(file: File, repo: Repository): boolean {
```

Comment:
I think we can clean up the API from this unneccessary parameter
```suggestion
  public isChunkedUploadNeeded(file: File): boolean {
```